api/DTOs/SupplierDTO.cs

```diff
@@ -0,0 +1,32 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DTO.Supplier;
+
+[ApiExplorerSettings(IgnoreApi = true)]
+public class SupplierRequest : BaseDTO
+{
+    [Required]
+    public string? Code { get; set; }
+    [Required]
```

Comment:
Geen required gebruiken in DTO's want de validation is hiervoor verantwoordelijk